src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -249,7 +252,19 @@ private CookieOptions BuildCookieOptions()
             }
         }
 
-        protected override async Task HandleSignInAsync(ClaimsPrincipal user, AuthenticationProperties properties)
+        public async Task SignInAsync(ClaimsPrincipal user, AuthenticationProperties properties)
+        {
+            if (user == null)
+            {
+                throw new ArgumentNullException(nameof(user));
+            }
+
+            properties = properties ?? new AuthenticationProperties();
+            await HandleSignInAsync(user, properties);
+            Logger.SignedIn(Scheme.Name);
```

Comment:
But I imagine you can just add a Logger.Log at the top of your method is the only change for you, and implement the interface method directly instead of HandleSignIn